app/objects/c_agent.py

```diff
@@ -21,15 +20,16 @@ def display(self):
                     last_seen=self.last_seen.strftime('%Y-%m-%d %H:%M:%S'),
                     sleep_min=self.sleep_min, sleep_max=self.sleep_max, executors=self.executors,
                     privilege=self.privilege, display_name=self.display_name, exe_name=self.exe_name, host=self.host,
-                    watchdog=self.watchdog, contact=self.contact)
+                    watchdog=self.watchdog, contact=self.contact, c2=list(self.c2))
 
     @property
     def display_name(self):
         return '{}${}'.format(self.host, self.username)
 
     def __init__(self, sleep_min, sleep_max, watchdog, platform='unknown', server='unknown', host='unknown',
                  username='unknown', architecture='unknown', group='red', location='unknown', pid=0, ppid=0,
-                 trusted=True, executors=(), privilege='User', exe_name='unknown', contact='unknown', paw=None):
+                 trusted=True, executors=(), privilege='User', exe_name='unknown', contact='unknown', c2=['http'],
```

Comment:
couple of things here:

1) i'm not sure we want the agent to be initialized with a list of potential C2s. this would infer that the agent is sending that information back to the server on it's first beacon, which i don't think is necessary. instead, i think we should do this based on the type of agent it is using. the best way to do this (i think), is to create the c_agent when it is downloaded instead of on its first beacon (what we do today). then, through the file_svc:special_payloads functionality, we can add the contacts to the c_agent per agent (54ndc47, manx, ragdoll, etc.). 

the tricky part with this, is ensuring this design can be applied to both 54ndc47 agents, along with any other future agent, without modifying any server code - and if possible, no agent code (or minimal agent code).  

2) on c_agent currently, we have a "contact" property, which is the current contact point (html, tcp, udp, websocket, etc.) the agent is using. because 'c2' is replicating this, only with the potential contacts, not the current one, we should use the terminology "contacts" instead of "c2".